src/BL/CurrecytManager.java

```diff
@@ -0,0 +1,51 @@
+package BL;
+
+import java.util.HashMap;
+
+public class CurrencyManager {
+    //Определяет статическую константу - значение валюты по умолчанию
+    public static final CurrencyTitle defaultCurrency = CurrencyTitle.EUR;
+    // Определяет приватное поле data, предназначенное для хранения экземпляра класса CurrencyExchangeHandler
+    private final CurrencyExchangeHandler data;
+
+    // Конструктор
+    public CurrentManager(String rateFilePath) {
+        this.data = new CurrencyExchangeHandler(rateFilePath);
```

Comment:
here we should not take the file path as an argument. at the business layer, a class can store an address. and it’s better to put this address in a new constant above

Reply:
done